pkg/cmd/open.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pior/dad/pkg/helpers"
+	"github.com/pior/dad/pkg/project"
+)
+
+var openCmd = &cobra.Command{
+	Use:   "open [github]",
+	Short: "Open a link about your project",
```

Comment:
"Open a link to your project" would be a better phrasing I think.

Reply:
It's appropriate for the github page, but the command is supposed to open all types of links (like *Dev*, with a `open` section in dev.yml)